### PR TITLE
Redirects: Move Installing TYPO3 chapter to TYPO3 Explained

### PR DESCRIPTION
**Pull request by linawolf**
See https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted/pull/477

### CODE REVIEW

File: config/nginx/redirects.conf [hunk 1]
```diff
@@ -67,15 +67,15 @@ location = /Home/Extensions.html {
 # Add short links
 
 location = /installation {
-    return 303 /m/typo3/tutorial-getting-started/main/en-us/Installation/Index.html;
+    return 303 /m/typo3/reference-coreapi/main/en-us/Administration/Installation/Install.html
```
Comment (andreaskienast) [inline config/nginx/redirects.conf:70]: Trailing `;` is missing

File: config/nginx/redirects.conf [hunk 2]
```diff
@@ -67,15 +67,15 @@ location = /Home/Extensions.html {
...
 }
 
 location = /upgrade {
     return 303 /m/typo3/guide-installation/main/en-us/;
 }
 
 location = /system-requirements {
-    return 303 /m/typo3/guide-installation/main/en-us/In-depth/SystemRequirements/Index.html;
+    return 303 /m/typo3/reference-coreapi/main/en-us/Administration/Installation/SystemRequirements/Index.html
```
Comment (andreaskienast) [inline config/nginx/redirects.conf:78]: Trailing `;` is missing

File: config/nginx/redirects.conf [hunk 3]
```diff
@@ -881,12 +881,21 @@ location ~ ^/typo3cms/extensions/externalimport_tut/(master|latest|stable)/(.*)
 }
 
 # getting started rewrite 2024
+location ~ ^/m/typo3/tutorial-getting-started/(main|13.4|12.4)/en-us/Installation/TutorialDdev.html {
+   return 303 https://docs.typo3.org/m/typo3/tutorial-getting-started/$1/en-us/Installation/Install.html
```
Comment (andreaskienast) [inline config/nginx/redirects.conf:885]: Trailing `;` is missing

**Review by andreaskienast — CHANGES_REQUESTED** [review]

File: config/nginx/redirects.conf (hunk 1 above)
Comment (linawolf) [inline config/nginx/redirects.conf:70]:
```suggestion
    return 303 /m/typo3/reference-coreapi/main/en-us/Administration/Installation/Install.html;
```

File: config/nginx/redirects.conf (hunk 2 above)
Comment (linawolf) [inline config/nginx/redirects.conf:78]:
```suggestion
    return 303 /m/typo3/reference-coreapi/main/en-us/Administration/Installation/SystemRequirements/Index.html;
```

File: config/nginx/redirects.conf (hunk 3 above)
Comment (linawolf) [inline config/nginx/redirects.conf:885]:
```suggestion
   return 303 https://docs.typo3.org/m/typo3/tutorial-getting-started/$1/en-us/Installation/Install.html;
```

Comment (linawolf) [reply config/nginx/redirects.conf]: No idea I am not good in doing server thingies, I just copy what is there and try to adjust ist.

Comment (andreaskienast) [reply config/nginx/redirects.conf]: Ack

**Review by andreaskienast — APPROVED** [review]

**Pull request merged**